bundle/Resources/translations/ezrepoforms_policies.en.xlf

```diff
@@ -0,0 +1,86 @@
+        <note>key: policy.limitation.identifier.group</note>
+      </trans-unit>
+      <trans-unit id="cee78b25ec2de77cf09109e6a2ac175a515a8350" resname="policy.limitation.identifier.language">
+        <source>Language</source>
+        <target>Language</target>
+        <note>key: policy.limitation.identifier.language</note>
+      </trans-unit>
+      <trans-unit id="ab9163a435d32610a7a1af8df07ab38eebc4c7c6" resname="policy.limitation.identifier.newsection">
+        <source>New Section</source>
+        <target>New Section</target>
+        <note>key: policy.limitation.identifier.newsection</note>
+      </trans-unit>
+      <trans-unit id="a9958f1d76f4685bf5fc32eee7514046db942160" resname="policy.limitation.identifier.newstate">
+        <source>New State</source>
+        <target>New State</target>
+        <note>key: policy.limitation.identifier.newstate</note>
+      </trans-unit>
+      <trans-unit id="be2b0449251f53db814dc4fbbb27e534419add2b" resname="policy.limitation.identifier.node">
+        <source>Location</source>
+        <target>Location</target>
+        <note>key: policy.limitation.identifier.node</note>
+      </trans-unit>
+      <trans-unit id="9daf8e1a94a6409f9b9b102e042c2858e4307ea8" resname="policy.limitation.identifier.owner">
+        <source>Owner</source>
+        <target>Owner</target>
+        <note>key: policy.limitation.identifier.owner</note>
+      </trans-unit>
+      <trans-unit id="e3773b71c8e7fda97f221ab25a6c25164d179d84" resname="policy.limitation.identifier.parentclass">
+        <source>Parent Content Type</source>
+        <target>Parent Content Type</target>
+        <note>key: policy.limitation.identifier.parentclass</note>
+      </trans-unit>
+      <trans-unit id="39bf0d9c1e0e8b64193bf09d83fbd89c2dac3421" resname="policy.limitation.identifier.parentdepth">
+        <source>Parent Depth</source>
+        <target>Parent Depth</target>
+        <note>key: policy.limitation.identifier.parentdepth</note>
+      </trans-unit>
+      <trans-unit id="cb52ff8582459d1bc981e9f98cf1c79876f83a1e" resname="policy.limitation.identifier.parentgroup">
+        <source>Parent Group</source>
+        <target>Parent Group</target>
+        <note>key: policy.limitation.identifier.parentgroup</note>
+      </trans-unit>
+      <trans-unit id="6208b97847450b255559e4a56c51ff06b76a08dd" resname="policy.limitation.identifier.parentowner">
+        <source>Parent Owner</source>
+        <target>Parent Owner</target>
+        <note>key: policy.limitation.identifier.parentowner</note>
+      </trans-unit>
+      <trans-unit id="bb77b27363dad566197a200290f0f0b18baa4705" resname="policy.limitation.identifier.section">
+        <source>Section</source>
+        <target>Section</target>
+        <note>key: policy.limitation.identifier.section</note>
+      </trans-unit>
+      <trans-unit id="548bc922c693c8259cbfd3354451a3035e2fa6f6" resname="policy.limitation.identifier.siteaccess">
+        <source>Site Access</source>
```

Comment:
`SiteAccess` is better for consistency with the documentation: https://doc.ezplatform.com/en/1.11/guide/siteaccess/

Reply:
/cc @DominikaK

Reply:
Yes, please, SiteAccess - one word, two capitals ;)